src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -43,12 +53,10 @@ public class ExactCountConstraint : PrefixConstraint
         public ExactCountConstraint(int expectedCount, IConstraint itemConstraint)
             : base(itemConstraint)
         {
-            this.expectedCount = expectedCount;
-            this.DescriptionPrefix = expectedCount == 0
-                ? "no item"
-                : expectedCount == 1
-                    ? "exactly one item"
-                    : string.Format("exactly {0} items", expectedCount);
+            Guard.ArgumentNotNull(itemConstraint, "itemConstraint");
```

Comment:
@rprouse  I proposed and made the decision for the gui because it's "my" project. I didn't deign to propose it for the framework. :smile:

Seriously, I was __hoping__ that such an exercise could be something to help get the incipient core team activated. I suppose somebody has to create a proposal and shepherd it through them if that's going to happen. I think the standard should allow for differences in individual projects that need to preserve more backward compatibility - for example if we started working again on CF - but that individual projects should need to get permission to use a higher level than the standard

That said, as far as I can see, the framework is the project that would have the most chance of needing backward compatibility.